### ISSUE TITLE
Warning: Tailwind CSS is not installed wrong in monorepo with Oxlint

### ISSUE THREAD

**Issue by danciudev** [schoero/eslint-plugin-readable-tailwind#374]
### Description

I updated from version 4.4.0 to 4.5.0 in my monorepo, and it doesn't seem to be finding the tailwindcss file.

I have an internal package that contains the Tailwind configuration (theme package), and I link to it in each oxlint.config.ts file using the configs.

### Flavor

TSX

### Code Input

```tsx
-
```

### Expected Behavior

```tsx
Work all rules
```

### Actual Behavior

```tsx
2026-05-02T14:49:39.425138Z  INFO oxc_language_server::backend: oxlint version: 1.62.0

2026-05-02 16:49:43.556 [info] ⚠️ Warning: Tailwind CSS is not installed. Disabling rule better-tailwindcss/enforce-canonical-classes.

2026-05-02 16:49:43.557 [info] ⚠️ Warning: Tailwind CSS is not installed. Disabling rule better-tailwindcss/no-unnecessary-whitespace.

2026-05-02 16:49:43.557 [info] ⚠️ Warning: Tailwind CSS is not installed. Disabling rule better-tailwindcss/no-duplicate-classes.

2026-05-02 16:49:43.557 [info] ⚠️ Warning: Tailwind CSS is not installed. Disabling rule better-tailwindcss/no-conflicting-classes.

2026-05-02 16:49:43.557 [info] ⚠️ Warning: Tailwind CSS is not installed. Disabling rule better-tailwindcss/enforce-logical-properties.

2026-05-02 16:49:43.559 [info] ⚠️ Warning: Tailwind CSS is not installed. Disabling rule better-tailwindcss/no-unknown-classes.

2026-05-02 16:49:43.559 [info] ⚠️ Warning: Tailwind CSS is not installed. Disabling rule better-tailwindcss/no-restricted-classes.

2026-05-02 16:49:43.560 [info] ⚠️ Warning: Tailwind CSS is not installed. Disabling rule better-tailwindcss/no-deprecated-classes.
```

### ESLint Log

```shell
-
```

### ESLint Config

```typescript
import { resolve } from "node:path";
import { defineConfig } from "oxlint";

import { oxlintReactAppConfigFn } from "@redslim/oxlint";

const CONFIG = {
	tailwindcss: {
		entryCssFile: resolve(import.meta.dirname, "../../packages/theme/src/frameworks/tailwindcss/core.css"),
	},
};

export default defineConfig({
	...oxlintReactAppConfigFn(CONFIG),
	ignorePatterns: ["**/mockServiceWorker.js", "coverage/**", "node_modules/**"],
});
```

### Versions

- Oxlint: 1.62.0
- Parser: 
- Plugin: 
- Node: 24.x

**Comment by schoero** [comment]
The update from [v4.4.0 to v4.5.0](https://github.com/schoero/eslint-plugin-better-tailwindcss/compare/v4.4.1...v4.5.0) did fix an issue that prevented the warning to be shown previously.
The logic to locate the tailwindcss installation was not changed. 

The plugin likely couldn't find the tailwindcss package prior to v4.5.0 but failed to show you. 

What does your final config look like? I'm asking because `entryCssFile` is not a valid option, so there must be some kind of computation going on.

In a monorepo setup, you likely need to set the `cwd` if linting is started from the root of the repository. See [monorepo setup](https://github.com/schoero/eslint-plugin-better-tailwindcss#monorepo-setup).

**Comment by rahdirigs** [comment]
> In a monorepo setup, you likely need to set the `cwd` if linting is started from the root of the repository. See [monorepo setup](https://github.com/schoero/eslint-plugin-better-tailwindcss#monorepo-setup).

We are running into a similar issue with this setting. Our setup is such that we have an eslint-config package in our monorepo which exposes an API with several options and based on the options, a single flat config is generated (more or less similar to @antfu/eslint-config).
Now as eslint v10 supports nearest config resolution / lookup (v9 also supported this with a flag; dont remember what it was), we have an eslint.config.ts in each package. So it supports both linting from the package directory as well as the root. That's where resolving this cwd becomes tricky for when the eslint command is run from the root, because I have two cases now, when running from the package directory, I want to run it with `cwd: '.'`, and when running from root `cwd: './packages/my-ui-library'` is needed.
